src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -14,7 +14,7 @@
 DEFAULT_RESOURCES = {"cpu": "8", "memory": "4Gi"}
 DEFAULT_IMG_NAME = "3.9-imaging-dev"
 DEFAULT_DAG_NAME = "bioimg-ingestion"
-_SUPPORTED_EXTENSIONS = (".tiff", ".tif", ".svs")
+_SUPPORTED_EXTENSIONS = (".tiff", ".tif", ".svs", ".ndpi")
```

Comment:
Possible follow-up item: we should list the supported extensions in tiledb-bioimg so we maintain in only one place.

Reply:
The idea behind listing them here is that we pass them in the "Path Listing UDF", this way we fail in the first node of ingestion. If we allow the lister to parse all extension and move the validation for the supported file extensions to tiledb-bioimg this will lead the `Ingestion` stage to fail with `unsupported type`. I am ok with that if we agree I can create a ticket for this one.

Reply:
No change for now, just keep in mind for the future - eventually we'll need "is this supported" and "give me supported extension" functions in -bioimg.